### DISCUSSION TITLE
icon

### DISCUSSION THREAD

**Discussion by VladimirPVV** [sumatrapdfreader/sumatrapdf#2889] [General]
can i change the Sumatra icon?

**Comment by GitHubRulesOK** [comment]
many icon tools can change the exe, but then it will be unsigned, and subject to mupdf license, safest is compile and sign your own, but why change it ?

**Comment by kjk** [comment]
Yes, you can use a number of programs to change an icon in a windows .exe (including SumatraPDF.exe)

Some programs: https://listoffreeware.com/free-software-change-program-icon/
